src/components/Locations.js

```diff
@@ -13,7 +13,8 @@ export default function Locations() {
     "Austin, TX",
     "San Francisco, CA",
     "Burlington, VT",
-    "Oshkosh, WI"
+    "Oshkosh, WI",
+    "Atlanta, GA"
```

Comment:
```suggestion
    "Oshkosh, WI"
```
Hold off on adding stuff here for now. We'll add it after registrations for organizers close.